.gitignore

```diff
@@ -6,3 +6,4 @@ _test/test
 _test/echo_server
 _test/tmp
 _vendor*
+.idea/
```

Comment:
Do we need to add this for now? Trying to minimize the number of changes from the original library

Reply:
nvm, missed your comment in PR description that we are simply copying the commit from original repo.

Reply:
Actually, I added that :)

Reply:
nvm, It exists also in the original repo :p

Reply:
So I guess let's remove it to minimize changes

Reply:
https://github.com/quickfixgo/quickfix/blob/a3c36054c04c683f9d8193b8eefc2595395106ec/.gitignore#L4